### DISCUSSION TITLE
get_content and entries fieldtype

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#5198] [Troubleshooting] [answered]
I'm using the `get_content` tag, works as expected. However I'm trying to pull in an `entires` fieldtype and it's not working?

On my `home page` I'm using this:
```
{{ get_content from="/our-projects" }} 
    {{ title }}
{{ /get_content }}
```

On `/our-projects` I've got an entires fieldtype
```
{{ featured_projects }}
    {{ project_location }}
{{ /featured_projects }}
```

I'd like to use that on the` home page` too, but using it as above doesn't work?

I can see in the `get_content` docs, it says

> If you're using a fieldtype like entries to select which entries you'd like to render, then you don't even need this tag

Is there an additional step to get this working? Running 3.2.32

UPDATE
----
This works... Even though the docs say the `get_content` tag isn't required to make this work.
```
{{ get_content from="/our-projects" }} 
   {{ featured_projects }}
      {{ project_location }}
   {{ /featured_projects }}
{{ /get_content }}
```
Have I misunderstood the docs?

**Comment by byron-roots** [comment, marked as answer]
If in you home page blueprint you had an Entries fieldtype, you can call the field directly. e.g if your fieldtype was called project you could do the follow.
```
{{ project }}
    {{ featured_projects }}
        {{ project_location }}
    {{ /featured_projects }}
{{ /project }}
``` 
As you are fetching from another page using a url instead of a fieldtype you still need the `{{ get_content }}` tag.

**Reply by stephenmeehanuk** [reply]
Thanks, I must have misread the docs. Cheers.
